catroid/src/androidTest/java/org/catrobat/catroid/uitest/content/brick/PhiroSeekBarColorBrickTest.java

```diff
@@ -96,7 +98,7 @@ private void testFormulaEditorInput(int phiroLedColorEditText, int phiroLedColor
 		solo.clickOnView(solo.getView(phiroLedColorEditText));
 		solo.clickOnView(solo.getView(phiroLedColorTextView));
 
-		UiTestUtils.insertDoubleIntoEditText(solo, 100.0);
+		insertDoubleIntoEditText(solo, 100.0);
```

Comment:
Hmm, did pmd really complain about this? If I undo your changes here, I still don't get any pmd warnings.
If this isn't something pmd complains, please undo :smirk_cat: